### DISCUSSION TITLE
Error en el 'default_rng'

### DISCUSSION THREAD

**Discussion by semilun4** [LaunchX-InnovaccionVirtual/CursoIntroPython#77] [Q&A] [answered]
He avanzado con mis Katas en **Google Colab**, pero hoy que empecé a pasarlo a **VSC** no he podido resolver este error, ¿alguien lo tuvo y lo pudo solucionar?

![Captura de pantalla 2022-02-10 011545](https://user-images.githubusercontent.com/90874653/153356866-5fd3cc8f-f781-433a-8f83-89db595417c7.png)

**Comment by Dfredude** [comment, marked as answer]
Lo mas probable es que necesites actualizar tu version de numpy. Corre esto dentro de una celda anterior a la que te esta dando problemas:
```
%pip install numpy
%pip install --upgrade numpy
```
Despues reinicia tu **kernel** o **VS Code**

**Comment by doguedogue** [comment]
1. Lo primero es que tengas el kernel
2. La extensión de jupiter
3. Ejecutar la instalación de numpy y matplotlib

![image](https://user-images.githubusercontent.com/23409026/153359747-4ed23a81-637c-4d73-8f81-c8ee043c429d.png)
![image](https://user-images.githubusercontent.com/23409026/153359759-c2a334e6-2929-4bfa-8b6b-ba2ae1839603.png)

**Comment by semilun4** [comment]
Muchas gracias, tomé en cuenta lo que me comentaron, aunque lo único que funcionó fue desinstalar e instalar Python para tener la versión 3.10 😅.
